laravel-app/routes/web.php

```diff
@@ -14,7 +14,7 @@
 */
 
 Route::get('/', function () {
-    return view('welcome');
+    return redirect()->route('login');
 });
```

Comment:
Create LoginController to it. Do not expose the backend process in web routes!

ref:
https://laravel.com/docs/10.x/controllers#main-content